unstructured/ingest/cli/cmds/astra.py

```diff
@@ -0,0 +1,61 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig, Dict
+from unstructured.ingest.connector.astra import AstraDBWriteConfig, SimpleAstraDBConfig
+
+
+@dataclass
+class AstraDBCliConfig(SimpleAstraDBConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--token"],
+                required=True,
+                type=str,
+                help="Astra DB Token with access to the database.",
+            ),
+            click.Option(
+                ["--api_endpoint"],
```

Comment:
think this should be --api-endpoint. Click converts to api_endpoint later in the process.

Reply:
Yep, definitely. Updated this just now.